src/app/rendering.js

```diff
@@ -8,6 +8,11 @@ export function render({ component, inside, withClasses }) {
   }
   const componentToRender = withClassList(component, withClasses);
   inside.appendChild(componentToRender);
+
+  componentToRender.destroy = () => {
```

Comment:
Też nie miałem lepszej nazwy na to wtedy. Ale może po prostu removeFromParent() :) ? 
Bo to destroy troche mylace. Bo jednak nie niszczy, a mozemy to potem ewentualnie gdzies jeszcze przypisac. 
Jak uważacie.

Reply:
Poprawię